### PR TITLE
feat(ui5-radio-button): add accessibleNameRef property

### PR DESCRIPTION
**Pull request by dimovpetar**
Related to #4501

### CODE REVIEW

File: packages/main/src/RadioButton.js [hunk 1]
```diff
@@ -186,6 +186,18 @@ const metadata = {
 			type: String,
 		},
 
+		/**
+		 * Defines the IDs of the elements that label the component.
+		 *
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @public
+		 * @since 1.0.3
```
Comment (fifoosid) [inline packages/main/src/RadioButton.js:195]: since 1.1.0

File: packages/main/src/RadioButton.js [hunk 2]
```diff
@@ -174,8 +175,7 @@ const metadata = {
 		},
 
 		/**
-		 * Defines the text alternative of the component.
-		 * If not provided a default text alternative will be set, if present.
+		 * Defines the accessible name of the component.
```
Comment (fifoosid) [inline packages/main/src/RadioButton.js:178]: I am not sure that this is self explanatory enough

**Review by fifoosid — CHANGES_REQUESTED** [review]

File: packages/main/src/RadioButton.js (hunk 1 above)
Comment (dimovpetar) [reply packages/main/src/RadioButton.js]: Done

File: packages/main/src/RadioButton.js (hunk 2 above)
Comment (dimovpetar) [reply packages/main/src/RadioButton.js:178]: I have taken this sentence from the other components.  The second sentence "If not provided a default text ..." is not completely true, so I removed it.
I think we can add something like in Openui5 - https://openui5.hana.ondemand.com/api/sap.m.RadioButton#associations **"see WAI-ARIA attribute aria-label"**. But we should do this for all components or not at all. What do you think?

Comment (elenastoyanovaa) [reply packages/main/src/RadioButton.js:178]: It is explanatory enough. What is more, additional information could be found in the accessibility documentation.

**Review by elenastoyanovaa — APPROVED** [review]

**Review by fifoosid — APPROVED** [review]

**Pull request merged**